infinex-omnibus/staging/infinex-omnibus-test.ethereum.toml

```diff
@@ -0,0 +1,28 @@
+#  pnpm cannon build infinex-omnibus/staging/infinex-omnibus-test.ethereum.toml --chain-id 1 --dry-run
+#  pnpm cannon publish infinex-omnibus-test:latest@omnibus --chain-id 1 --exclude-cloned
+
+name = "infinex-omnibus-test"
+version = "3.0.0"
+preset = "omnibus"
+description = "All Infinex Deployments"
+
+[pull.AccountFactoryV1]
+source = "infinex-account-factory-test:1.0.0@main"
+
+[pull.CoreV1]
+source = "infinex-multichain:1.0.0staging@O2"
+
+[pull.CoreV2]
+source = "infinex-multichain:1.1.0staging@O2"
+
+[pull.CoreV3]
+source = "infinex-core-test:3.0.0@core"
+
+[pull.CurveStableSwapV1]
+source = "infinex-multichain:1.0.0staging@apps"
+
+[pull.CurveStableSwapV2]
+source = "infinex-curve-app-test:1.1.0@curve-stableswap"
+
+[pull.PatronPurchaseAppV1]
+source = "infinex-patron-test:1.0.0@patron-purchase"
```

Comment:
same here

Reply:
@lemoninfinex 1.0.0 for staging is right